### ISSUE TITLE
ISA: document title question

### ISSUE THREAD

**Issue by dthaler** [ebpffoundation/ebpf-docs#89]
The Internet Draft filename is draft-ietf-bpf-isa-XX, and the charter has:
> [PS] the BPF instruction set architecture (ISA) that defines the 
> instructions and low-level virtual machine for BPF programs,

That is, "instruction set architecture (ISA)", but the document itself has:
```
=======================================
BPF Instruction Set Specification, v1.0 
=======================================

This document specifies version 1.0 of the BPF instruction set.
```

Notably, no "architecture (ISA)".   Also, we now have a mechanism
to extend it with conformance groups over time, so "v1.0" seems less relevant and perhaps not important given there's only one version being standardized at present.

What about changing the doc to say:
```
=======================================
BPF Instruction Set Architecture
=======================================

This document specifies the BPF instruction set architecture (ISA).
```
?

**Comment by dthaler** [comment]
Question posted to list at https://mailarchive.ietf.org/arch/msg/bpf/SEpn3OL9TabNRn-4rDX9A6XVbjM/

**Comment by dthaler** [comment]
Patch posted: https://mailarchive.ietf.org/arch/msg/bpf/NeURxAIvEURLIlHWMx08u9xetTc/

**Comment by dthaler** [comment]
Fixed in #99

**Comment by dthaler** [comment]
Fixed in draft -01
